app-modules/authorization/database/factories/RoleFactory.php

```diff
@@ -48,7 +48,7 @@ public function definition(): array
     {
         return [
             'name' => fake()->text(25),
-            'guard_name' => fake()->randomElement(['web', 'api']),
+            'guard_name' => 'web',
```

Comment:
Did you make this change because you need some seeder to be `web`?

If that is the case, let's leave this as the `randomElement` generation and create two factory states that set this as `web` or `api` and use that where you require it to be `web`

Reply:
The guard can't be api anymore because of the `config/auth.php` change to the api guard to be provided by `system-users`. Otherwise you get things like this:

```
 FAILED  Appmodules\authorization\tests\Feature\Actions\AddNewRoleToExistingUsersInRoleGroupTest > it will not overwrite an existing role that was directly assigned to the user              GuardDoesNotMatch   
  The given role or permission should use guard `web` instead of `api`.
```